SPONSORSHIP.md

```diff
@@ -0,0 +1,68 @@
+# Sponsorship
+
+An obvious way to contribute to the Flux project is to spend time on
+things that need to be done. But not everyone can set aside time that
+way as an individual, or commit team members' time as an organisation.
+
+We would like to recognise contributions of money and material (e.g.,
+cloud platform credits), whether it is directly to the project or
+through a sponsorship program. And we would like to let people know
+there are ways to support the Flux project other than devoting time
+and effort directly.
+
+This document describes how the sponsorship part of that works.
+
+A sponsorship program is a way for organisations to contribute money
+(or a near equivalent, e.g., cloud platform credits) to help the Flux
+project. The organisers of the program use the money to pay for things
+that directly benefit the Flux project, for instance paying employees
+that work as Flux maintainers.
+
+The Flux project does not run its own sponsorship program, but other
+people can, and we vet those who wish to be recognised. The programs
+are listed on the Flux website, and sponsors are recognised on the
+website amongst other kinds of contributions.
+
+## Recognition of sponsorships
+
+The Flux website shows contributions made via a sponsorship
+program. The Flux maintainers agree a reasonable wording with the
+organisers of a sponsorship program. For example, a program might have
+different levels of contributions ("gold", "silver", "bronze"), and to
+recognise larger sponsorships, the program organisers would want the
+level to be mentioned.
+
+The maintenance of active sponsorships is done through pull requests
+to the Flux website repository. It's up to sponsorship program
+organisers to make sure their sponsors are represented accurately.
+
+## Sponsorship programs
+
+It's important to establish trust between the Flux community and the
+organisers of a sponsorship program. Mainly, we need to check that
+sponsors' money will be put to the benefit of the project.
+
+There is an initial exchange in which the program organisers and the
+Flux maintainers agree what benefits are being provided and how
```

Comment:
I wonder whether we need or should weave this into the GOVERNANCE.md document so that sponsorship is more tied into the overall governance of the project and doesn't live a life in isolation from everything else.

The least this document should do is make clear what it means by "Flux maintainers". I suppose this should mean [core maintainers](https://github.com/fluxcd/community/blob/main/community-roles.md#core-maintainers).

Reply:
@makkes do you agree we should include all Flux maintainers - automated from [the source of truth here](https://github.com/fluxcd/community/blob/main/project/flux-project-maintainers.yaml) - not only core (fluxcd/flux2)?

Reply:
Yes I do.